packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-case-sign-in-soc-idp/main/index.md

```diff
@@ -1,10 +1,10 @@
 ---
-title: Sign in with Facebook using the Widget
+title: Use the Sign-In Widget with Facebook
```

Comment:
this change can be a bit deceptive and moves it into a grey area of understanding LOL.  I fixed the gerund issue and you shouldn't get dinged now if you change this back.